TM1py/Services/CellService.py

```diff
@@ -151,14 +154,15 @@ def relative_proportional_spread(
         :param unique_element_names: target cell coordinates as unique element names (e.g. ["[d1].[c1]","[d2].[e3]"])
         :param reference_cube: name of the reference cube. Can be None
         :param reference_unique_element_names: reference cell coordinates as unique element names
+        :param sandbox_name: str
         :return:
         """
         mdx = """
         SELECT
         {{ {rows} }} ON 0
         FROM [{cube}]
         """.format(rows="}*{".join(unique_element_names), cube=cube)
-        cellset_id = self.create_cellset(mdx=mdx, **kwargs)
+        cellset_id = self.create_cellset(mdx=mdx, sandbox_name=sandbox_name, **kwargs)
```

Comment:
Don't we have to reference the sandbox in the `_post_against_cellset` (line 179) call as well?